sdk/keyvault/azure-keyvault-certificates/CHANGELOG.md

```diff
@@ -1,20 +1,20 @@
 # Release History
 
-## 4.7.0b1 (Unreleased)
+## 4.7.0 (2023-03-15)
 
 ### Features Added
 - Added support for service API version `7.4`
 - Clients each have a `send_request` method that can be used to send custom requests using the
   client's existing pipeline ([#25172](https://github.com/Azure/azure-sdk-for-python/issues/25172))
 
-### Breaking Changes
-
 ### Bugs Fixed
 - The type hints for `KeyVaultCertificate.cer` and `DeletedCertificate.cer` are now
   `Optional[bytearray]` instead of `Optional[bytes]`
   ([#28959](https://github.com/Azure/azure-sdk-for-python/issues/28959))
 
 ### Other Changes
+- Python 3.6 is no longer supported. Please use Python version 3.7 or later.
```

Comment:
Shouldn't this be a breaking change?

Reply:
(Same for anywhere else)

Reply:
It technically might be, but the Python SDK has been putting this notice in our "Other Changes" section for each library. (Possibly because the library itself isn't introducing a breaking change? Not sure)